src/app/settings/page.tsx

```diff
@@ -1,29 +1,110 @@
 'use client';
 
+import React from 'react';
 import { useRouter } from 'next/navigation';
-import supabase from '../../api/supabase/createClient';
+import supabase from '@/api/supabase/createClient';
+import { BackLink, H1, H4 } from '@/styles/text';
+import Button from '@/components/Button';
+import COLORS from '@/styles/colors';
+import SettingsSection from '@/components/SettingsSection';
+import { ContentContainer, PageContainer } from './styles';
 
 export default function Settings() {
   const { push } = useRouter();
+
   const handleSignOut = async () => {
     const { error } = await supabase.auth.signOut();
     if (error) {
       throw new Error(`An error occurred trying to sign out: ${error.message}`);
     }
     push('/login');
   };
-  const resetPassword = () => {
-    push('/reset-password');
-  };
+
+  // const resetPassword = () => {
+  //   push('/reset-password');
+  // };
 
   return (
-    <>
-      <button type="button" onClick={handleSignOut}>
-        Sign out
-      </button>
-      <button type="button" onClick={resetPassword}>
-        Reset Password
-      </button>
-    </>
+    <PageContainer>
+      <BackLink href="/cases">Back</BackLink>
+      <ContentContainer>
```

Comment:
@davidqing6432 curious if we can move the entire settings column up so it's closer to the top of the page? i.e. at the level of the back button maybe

they feel pretty far away from each other rn, which is chill but my only concern is that you can scroll a tiny bit and the screen still looks normal but the back button is gone

<img width="1512" alt="Screenshot 2023-11-23 at 8 13 13 PM" src="https://github.com/calblueprint/immigration-justice-project/assets/47801900/bfadba14-90d5-41b7-8f5b-4ec5cea0cb65">

Reply:
yeah we can move it up!!